specification/web/resource-manager/Microsoft.Web/stable/2020-10-01/examples/GetWebAppStacks.json

```diff
@@ -0,0 +1,2915 @@
+{
+  "parameters": {
+    "api-version": "2020-10-01"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "value": [
+          {
+            "id": null,
```

Comment:
Please clean up all the "stacks" examples.  "id" cannot be null and the "type" should not contain the query string

Reply:
The "id" property was removed instead of populated with the correct value.  "id" is a require property and must be populated with the resource ID

Reply:
I noticed that id got removed in all the new example files, please fix this as well.

Reply:
I have added the "id" property back, populated with resource ID